src/main/resources/application.properties

```diff
@@ -2,25 +2,25 @@ quarkus.banner.enabled=false
 quarkus.package.type=uber-jar
 
 # Debezium configuration
-gingersnap.rule.us-east.connector.connector=MYSQL
-gingersnap.rule.us-east.connector.schema=debezium
-gingersnap.rule.us-east.connector.table=customer
+%dev.gingersnap.rule.us-east.connector.connector=MYSQL
```

Comment:
Why are the property changes required?

Reply:
When the test starts, it uses this properties file but runs using the test profile. Without adding the dev profile, the test will try to connect to the database/backend and fail. Looking at their docs, I didn't find a way to override the properties as a whole, only by creating a class that explicitly overrides specific keys.

Reply:
If using the quarkus CLI or the maven wrapper during development, everything should work the same way. The only difference is running the jar needs to pass the `Dquarkus.profile=dev`.

Reply:
Makes sense, thanks for explaining.